lib/streamlit/elements/form.py

```diff
@@ -246,12 +251,17 @@ def form_submit_button(
             True if the button was clicked.
         """
         ctx = get_script_run_ctx()
+        type = streamlit.elements.button.check_valid_button_type(
```

Comment:
Feedback appreciated as I'm not sure this is the proper way to import/call the `check_valid_button_type` function 🤔 

In `form.py` importing as such:
```python
from streamlit.elements.button import check_valid_button_type
```
causes a circular import error (since this button.py file imports the `current_form_id, is_in_form` functions from form.py).

Reply:
Hmm, I think in this case I don't think it is worth having a shared method. I would just duplicate this code for the commands, e.g.:

```python
if type not in ["primary", "secondary"]:
    raise StreamlitAPIException(
        'The type argument to st.form_submit_button must be "primary" or "secondary". \n'
        f'The argument passed was "{type}".'
    )
```

In case you change the type to `Literal["primary", "secondary"]`, we probably also do not need to lowercase the string since it would already be an invalid value based on type checks.

Reply:
Makes sense, thank you!